### DISCUSSION TITLE
The Analyzers Greynoise and OTX are not working for IP addresses

### DISCUSSION THREAD

**Discussion by ghost** [Security-Onion-Solutions/securityonion#8183] [General] [answered]
I am getting invalid input for Greynoise and Internal analyzer failure for OTX. 
The Below message is for Greynoise.
{
  "response": {
    "message": "Request is not a valid routable IPv4 address"
  },
  "status": "caution",
  "summary": "invalid_input"
}

Is this expected?

**Comment by weslambert** [comment, marked as answer]
What kind of observable are you attempting to use with the analyzers? Are you able to share the value of the observable? They seem to think it's not a valid IP address.

**Comment by ghost** [comment]
Hi Wes,
The IP is 104.217.249.182.

**Comment by ghost** [comment]
Hi Wes,
I checked by adding another IP as Observable. The Analyzers are working fine. Both Greynoise and OTX are working. I think the problem was that I might have added a space while adding the observables. 
Thanks for the help!!
